pkg/account/handler.go

```diff
@@ -0,0 +1,167 @@
+package account
+
+import (
+	"container/list"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/labstack/echo/v4"
+)
+
+var account_list = list.New()
```

Comment:
You don't really have to get a mock like this. If you intend to use it as a test, try sqlite in-memory db provided by gorm.

Reply:
Our group's opinion is to implement the interface first, so I just use `list` for simple implementation. I will use DB later after we figure out the account table's detailed struct with other group : )